File: Project/Project.DAL/Models/Meeting.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

#nullable disable

namespace Project.DAL.Models
{
    public partial class Meeting
    {
        [Key]
        [Required]
        public int id { get; set; }
        public int idProf { get; set; }
        public int idUser { get; set; }
        [Required]
        public string title { get; set; }
        [Required]
        public DateTime date { get; set; }
        [Required]
        public TimeSpan time { get; set; }
        [Required]
        public string content { get; set; }
        [Required]
        public bool isApproved { get; set; }
        [ForeignKey("idUser")]
        public virtual User User { get; set; }
        [ForeignKey("idProf")]
        public virtual Expert Expert { get; set; }
    }
}

File: clienMYExpert/src/app/classes/meeting.ts
import { Time } from '@angular/common';

export class Meeting {
    id: number
    idProf: number;
    idUser: number;
    title: string;
    date: Date;
    time: string;
    content: string;
    isApproved: boolean;
    constructor(id?: number, profId?: number, userId?: number, title?: string, date?: Date, time?: string, content?: string, isApproved?: boolean) {
        this.id = id;
        this.idProf = profId;
        this.idUser = userId;
        this.title = title;
        this.date = date;
        this.time = time;
        this.content = content;
        this.isApproved = isApproved
    }
}
